integration-tests/bats/import-update-tables.bats

```diff
@@ -205,7 +205,7 @@ DELIM
     [ "$status" -eq 1 ]
     [[ "$output" =~ "A bad row was encountered while moving data" ]] || false
     [[ "$output" =~ "Bad Row:" ]] || false
-    [[ "$output" =~ '"123456" is not valid for column "c" (type "CHAR(5)")' ]] || false
+    [[ "$output" =~ 'string is too large for column' ]] || false
```

Comment:
This is a regression. The error message must include the offending value. It can also include the entire row.

Reply:
Added to the regex the entire row.